### PR TITLE
Closes #<1713> 1713 call convert blanks@devel

### PR DESCRIPTION
**Pull request by danleibovitz**

### CODE REVIEW

**Comment by bms63** [comment]
Hi @danleibovitz where is the PR template and is this ready to review?

**Review by bms63 — CHANGES_REQUESTED** [review]
Thanks for doing this PR.  Since there is only one function involved we should just do `dm <- convert_blanks_to_na(admiral_dm)` rather than `dm <- admiral_dm %>% convert_blanks_to_na()`

**Comment by bms63** [comment]
@zdz2101 I changed a few lines here - can you just double check real quick.

@danleibovitz we got it from here.  trying to get all the PRs done in the next few hours.

File: vignettes/pk_adnca.Rmd [hunk 1]
```diff
@@ -108,7 +108,7 @@ data("admiral_ex")
 data("admiral_pc")
 data("admiral_vs")
 
-adsl <- admiral_adsl
+adsl <- admiral_adsl %>% convert_blanks_to_na()
```
Comment (zdz2101) [inline vignettes/pk_adnca.Rmd:111]: To match consistency, remove this one too?

Comment (zdz2101) [inline vignettes/pk_adnca.Rmd:111]:
```suggestion
adsl <- admiral_adsl
```

**Review by zdz2101 — APPROVED** [review]

**Pull request merged**